Backpack/Alert/Classes/BPKAlertButtonConfiguration.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <Backpack/Button.h>
+
+typedef void (^BPKAlertButtonActionHandler)(void);
+
+NS_SWIFT_NAME(AlertButtonConfiguration) @interface BPKAlertButtonConfiguration : NSObject <NSCopying>
```

Comment:
Can we wrap this and all other files in `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END` as per the [descision](https://github.com/Skyscanner/backpack-ios/blob/a05e894d79806a8ab794b17099658cedb0ad8f2a/decisions/assume-non-null-by-default.md)

Reply:
👍